components/sso-saml/org.wso2.carbon.identity.sso.saml.ui/src/main/resources/web/sso-saml/add_service_provider_finish.jsp

```diff
@@ -89,9 +89,10 @@
         	serviceProviderDTO.setNameIDFormat(serviceProviderDTO.getNameIDFormat().replace(":", "/"));
         }
 
-        if ("true".equals(request.getParameter("enableAttributeProfile"))) {
+        if (request.getParameter("enableAttributeProfile") != null) {
```

Comment:
Fixes are in [1]. Also included the fixes for other string literals in the same page as well.
[1] https://github.com/wso2/carbon-identity/pull/894